README.md

```diff
@@ -119,6 +119,10 @@ aws:
     attribute_names:
       - ApproximateReceiveCount
       - SentTimestamp
+sqs:
+  log_level: debug
+sns:
+  log_level: info
```

Comment:
Sorry, I wasn't completely clear. This is on a system where I have configured the ActiveJob integration with the initializer listed here; https://github.com/phstc/shoryuken/wiki/Rails-Integration-Active-Job

e.g.
```
Shoryuken.configure_server do |config|
  Rails.logger = Shoryuken::Logging.logger
end
```

`info.log` and `debug.log` are rails logs, this is what they look like pre and post the change; https://gist.github.com/atyndall/b44127df03a4717a76cc6de50a5262f8

As you can see, SQS is very noisy.